src/osx/core/sound.cpp

```diff
@@ -134,6 +134,7 @@ bool wxSound::Create(const wxString& fileName, bool isResource)
 {
     wxCHECK_MSG( !isResource, false, "not implemented" );
 
+#ifndef __WXOSX_IPHONE__
     wxCFRef<CFURLRef> url(wxOSXCreateURLFromFileSystemPath(fileName));
```

Comment:
Again, I think it should be possible to provide `wxOSXCreateURLFromFileSystemPath()` for iOS instead.

Reply:
like in the metafile.cpp example, we need to know, why there  is a distinction between MACOS and OSX.